generator/04-syndesis-oauth-proxy.yml.mustache

```diff
@@ -0,0 +1,122 @@
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: syndesis
+      component: syndesis-oauthproxy
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: syndesis-oauthproxy-tls
+    name: syndesis-oauthproxy
+  spec:
+    ports:
+    - port: 8443
+      protocol: TCP
+      targetPort: 8443
+    selector:
+      app: syndesis
+      component: syndesis-oauthproxy
+- apiVersion: v1
+  kind: Route
+  metadata:
+    labels:
+      app: syndesis
+    name: syndesis
+  spec:
+    host: ${ROUTE_HOSTNAME}
+    port:
+      targetPort: 8443
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+      kind: Service
+      name: syndesis-oauthproxy
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: syndesis
+      component: syndesis-oauthproxy
+    name: syndesis-oauthproxy
+  spec:
+    replicas: 1
+    selector:
+      app: syndesis
+      component: syndesis-oauthproxy
+      deploymentconfig: syndesis-oauthproxy
+    strategy:
+      resources:
+        limits:
+          memory: "100Mi"
```

Comment:
@jimmidyson could you please increase this to 200Mi ? The reason is that the minimum container size must be 100Mi, and our production cluster does not allow the spec of `requests` but calculates them as 50% of `limits` (and still must be >= 100Mi)

Reply:
oops, meant you should add the limits to the container. This is for the deploy container (we used 256MB here, but I guess the same limit logic applies)

Reply:
Done